tests/mocha/mellowLpRouter.test.ts

```diff
@@ -1057,7 +1057,7 @@ describe('Mellow Router Test Suite', () => {
     });
 
     it('Calculate correct transaction fee in USD for autorollover registration', async () => {
-      const currentEthPrice = await geckoEthToUsd(process.env.REACT_APP_COINGECKO_API_KEY || '');
+      const currentEthPrice = await priceFetch.geckoEthToUsd(process.env.REACT_APP_COINGECKO_API_KEY || '');
```

Comment:
I don't like this in tests, we don't have env vars in pipeline so this will fail (when we add tests to pipeline).
try 
```
sinon.stub(priceFetch, 'geckoEthToUsd').callsFake(
      () =>
        new Promise((resolve, reject) => {
            setTimeout(() => {
              resolve(1300);
            }, 10);
        }),
    );
```

Reply:
actually I think this is already there. It gives value 1300. use that instead pls